lib/hanami/common_logger.rb

```diff
@@ -66,7 +66,7 @@ def log(env, status, header, began_at)
         path:    env[SCRIPT_NAME] + env[PATH_INFO].to_s,
         length:  length,
         params:  extract_params(env),
-        elapsed: now - began_at
+        elapsed: '%.4f' % (now - began_at)
```

Comment:
I may be wrong but the documentation in [this page](https://guides.hanamirb.org/projects/logging/) suggests that the logline shows the milliseconds/seconds at the end of it but performing some tests with a fresh new application I noticed that what we actually had was the following structure:

`[APP] [INFO] [2021-01-31 11:17:40 -0300] HTTP/1.1 GET 200 ::1 /users 175 {} 2021-01-27 11:39:43 -0300`
`We actualy had a DateTime information.`

Using the monotonic time I ended with a huge decimal value representing the elapsed time in milliseconds, that's the reason I decided to format, capping it to four decimals only. I supposed it is enough to make decisions based on the response time or even grab it and plot it in a dashboard but let me know any other different opinions, please.

`[APP] [INFO] [2021-01-31 11:17:40 -0300] HTTP/1.1 GET 200 ::1 /users 175 {} 0.0018`
`[APP] [INFO] [2021-01-31 11:17:40 -0300] HTTP/1.1 GET 200 ::1 /users 175 {} 0.0218`